docs/faq.md

```diff
@@ -29,6 +29,54 @@ For detail information of each config field, please run:
+2. If this is the case; go ahead and do a FRESH install of Spicetify + Spotify.
+3. If the version of Spotify inside of your `config-xpui.ini` does not correspond with what is actually installed, this could mean your config is out of sync. Delete your config file and run `spicetify` in a new terminal instance. If it’s still reflecting an incorrect entry, delete your "prefs" files inside of `C:\Users\YOUR_USERNAME\AppData\Roaming\Spotify\prefs`.
+4. Some themes have had histories of deleting the main view of Spotify by accident, make sure to troubleshoot across multiple themes and extensions.
+
+### GNU/Linux + MacOS
+
+1. Spicetify has a history of incompatibility with Spotify on GNU/Linux, start off by assuring you are not using snap's version of Spotify.
+2. It’s possible the regex is broken for the latest versions of Spicetify, unfortunately there is no fix for this, and you'll instead have to downgrade Spotify and Spicetify to ensure they work together.
+3. Ask around in the Spicetify discord server, there are lots of active GNU/Linux and macOS users ready to give you pointers.
+
+## Cannot use term "Spicetify" in my terminal.
+
+### Universal
+
+1. This is almost always an issue with your $path on GNU/Linux and mac, or your PATH on Windows.
+2. Google how to add the corresponding paths to your path based on your OS;
+  - **Windows**: `%USERPROFILE%/spicetify-cli` --> `C:\Users\tomfi\spicetify-cli`
+  - **GNU/Linux**: `$HOME/.spicetify`
+  - **MacOS**: `$HOME/.spicetify`
+
+## Why can’t I open devtools anymore?
+#### *Thanks to Spotify loving the ricing/modding community, in later versions of Spotify they have decided to REMOVE debugging tools such as inspect element. However remote debugging can bypass this limitation*.
+
+### Windows
+
+1. Start by finding your preferred Spotify shortcut. Right click this shortcut and enter properties, inside of the "target" text box, at the very end of the path, add a space and type in "--remote-debugging-port=9222". Apply and close.
+2. Navigate to Google Chrome and in the address tab type "chrome://inspect", click on the box that says 
+"port forwarding". In the new menu there will be a table of values, on a new line, in the first box input the port "9222" and the second box "localhost:9222", leave port forwarding DISABLED and press done.
+3. After that providing you have Spotify open; you should see it in the list of connected targets. Press the inspect button and voila!
+
+### GNU/Linux + MacOS
+1. Locate your Spicetify config using the command `spicetify config-dir` and enter `spotify_launch_flags = --remote-debugging-port=9222`
+2. Open google chrome and go to "chrome://inspect" and Spotify should pop up, from there press inspect.
+3. If Spotify does not pop up click on the box that says, "port forwarding". In the new menu there will be a table of values, on a new line, in the first box input the port "9222" and the second box "localhost:9222", leave port forwarding DISABLED and press done.
```

Comment:
Again its a very frequently asked question especially in the discord server

Reply:
Yes, because it is not anywhere... 😅 If we add it to the docs then people won't need to ask as much! Just because it is a FAQ doesn't mean that it belongs under FAQ, you feel me? ahaha

Reply:
Moved